app/containers/ViewQuestion/validate.js

```diff
@@ -83,14 +83,15 @@ export const postAnswerValidator = (
     message = `${translations[messages.alreadyAnswered.id]}`;
   } else if (
     questionData.user === profileInfo.user &&
-    profileInfo.ratings.get(communityId) < MIN_RATING_FOR_MY_QUESTION
+    profileInfo.ratings.get(communityId.toString()) < MIN_RATING_FOR_MY_QUESTION
```

Comment:
extract `profileInfo.ratings.get(communityId.toString())` to
```
const profileCommunityId = profileInfo.ratings.get(communityId.toString());
...
profileCommunityId < MIN_RATING_FOR_MY_QUESTION
..
profileCommunityId < MIN_RATING_FOR_OTHER_QUESTIONS
```
please update whole `validate.js` file